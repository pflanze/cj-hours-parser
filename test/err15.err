Error: can't process file "test/err15.txt"

Caused by:
    0: can't parse line 26, " Mon_31_Oct_200350_CET_2022-{Mon 31 Oct 21:18:04 GMT 2022}infrastructure"
    1: can't parse as a processing instruction: " Mon_31_Oct_200350_CET_2022-{Mon 31 Oct 21:18:04 GMT 2022}infrastructure"
    2: does not start with `\w+\s*=` nor `set \w+ `
