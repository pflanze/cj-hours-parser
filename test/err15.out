item=SingleLineComment(Owned(" this is a comment"))
    =SingleLineComment(Owned(" this is a comment"))
item=MultiLineComment([Owned("this is a multiline"), Inline("  comment")])
    =MultiLineComment([Owned("this is a multiline"), Inline("  comment")])
item=Other([Minus, Time(NT(12:44:21)), Rest(Inline("test  some time"))])
    =Other([Subitem(L(0, 6, 0), Minus), Subitem(L(0, 6, 1), Time(NT(12:44:21))), Subitem(L(0, 6, 10), Rest(Inline("test  some time")))])
item=Other([Minus, Time(NTZ(17:16:31, Zone(Inline("UTC")))), Rest(Owned("time for some stuff"))])
    =Other([Subitem(L(0, 7, 0), Minus), Subitem(L(0, 7, 1), Time(NTZ(17:16:31, Zone(Inline("UTC"))))), Subitem(L(0, 7, 13), Rest(Owned("time for some stuff")))])
item=Other([Minus, Time(NTZ(17:16:00, Zone(Inline("UTC")))), Rest(Owned("time for some stuff"))])
    =Other([Subitem(L(0, 8, 0), Minus), Subitem(L(0, 8, 1), Time(NTZ(17:16:00, Zone(Inline("UTC"))))), Subitem(L(0, 8, 10), Rest(Owned("time for some stuff")))])
item=Empty(1)
    =Empty(1)
item=MultiLineComment([Owned("some more commenting")])
    =MultiLineComment([Owned("some more commenting")])
item=SingleLineComment(Inline("[and yet more]"))
    =SingleLineComment(Inline("[and yet more]"))
item=MultiLineComment([Inline("blabla")])
    =MultiLineComment([Inline("blabla")])
item=ProcessingInstruction { key: Inline("TZ"), value: Inline("Zurich") }
    =ProcessingInstruction { key: Inline("TZ"), value: Inline("Zurich") }
item=Other([Time(NT(03:27:00)), Minus, Time(NT(06:44:01)), Rest(Inline("test-again"))])
    =Other([Subitem(L(0, 18, 0), Time(NT(03:27:00))), Subitem(L(0, 18, 5), Minus), Subitem(L(0, 18, 6), Time(NT(06:44:01))), Subitem(L(0, 18, 14), Rest(Inline("test-again")))])
item=ProcessingInstruction { key: Inline("TZ"), value: Inline("london") }
    =ProcessingInstruction { key: Inline("TZ"), value: Inline("london") }
item=Empty(2)
    =Empty(2)
item=ProcessingInstruction { key: Inline("hello"), value: Inline("123") }
    =ProcessingInstruction { key: Inline("hello"), value: Inline("123") }
item=Empty(1)
    =Empty(1)
item=ProcessingInstruction { key: Inline("hello2"), value: Inline("123too") }
    =ProcessingInstruction { key: Inline("hello2"), value: Inline("123too") }
item=Empty(1)
    =Empty(1)
